flash/utils/imports.py

```diff
@@ -5,3 +5,8 @@
 _COCO_AVAILABLE = _module_available("pycocotools")
 _TIMM_AVAILABLE = _module_available("timm")
 _TORCHVISION_AVAILABLE = _module_available("torchvision")
+try:
+    import transformers
+    _TRANSFORMERS_AVAILABLE = True
```

Comment:
Isn't this what `_module_available` does?

Reply:
Somehow, it doesn't work with transformers as spec isn't available.

Reply:
That's strange, `_module_available("transformers")` is giving true for me but I haven't tried it without transformers installed